src/Tribe/Admin/Timezone_Updater.php

```diff
@@ -84,7 +84,7 @@ public function notice_inner() {
 		}
 
 		if ( $progress >= 0 ) {
-			$percent = sprintf( __( '%d%% complete', 'the-events-calendar' ), $progress );
+			$percent = sprintf( _x( '%d%% complete', 'The "percent complete" of a process, e.g. "45% complete"', 'the-events-calendar' ), $progress );
```

Comment:
`esc_html_x()`
and should it be `%f` to allow floating point? I'm fine with `%d` but just checking (applies to other _percent complete_ strings in this PR as well)